src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Tree/Dependencies/DependenciesProjectTreeProvider.cs

```diff
@@ -37,35 +37,35 @@ internal class DependenciesProjectTreeProvider :
     {
         private readonly IDependencyTreeTelemetryService _treeTelemetryService;
         private readonly object _treeUpdateLock = new object();
-        private Task _treeUpdateQueueTask = null;
+        private Task _treeUpdateQueueTask;
 
         /// <summary>
         /// Initializes a new instance of the <see cref="DependenciesProjectTreeProvider"/> class.
         /// </summary>
         [ImportingConstructor]
         public DependenciesProjectTreeProvider(
             IProjectThreadingService threadingService,
-            UnconfiguredProject project,
+            UnconfiguredProject unconfiguredProject,
```

Comment:
@davkean didn't you have a preferred naming convention here to follow along with CPS?

Reply:
In this instance I made the parameter name match the base ctor's.